File: initiate/ios/sample.m
[self.hyperInstance initiate:self payload:initiationPayload callback:^(NSDictionary *data) {
      if (data && data[@"event"]) {
          if ([event isEqualToString:@"show_loader"]) {
              // Show some loader here
          } else if ([event isEqualToString:@"hide_loader"]) {
              // Hide Loader
          } else if ([event isEqualToString:@"initiate_result"]) {
              // Get the payload
              NSDictionary *payload = data["payload"];
          } else if ([event isEqualToString:@"process_result"]) {
              // Get the payload
              NSDictionary *payload = data["payload"];
              // Merchant handling
          }
      } else {
        //Error handling
     }
}];

File: initiate/ios/sample.swift
self.hyperInstance.initiate(self, payload: initiationPayload, callback: { [unowned self] (response) in
     if let data = response {
         let event = data["event"] as? String ?? ""
         if event == "show_loader" {
           // Show some loader here
         } else if event == "hide_loader" {
           // Hide Loader
         } else if event == "initiate_result" {
           // Get the payload
           let payload = data["payload"]
         } else if event == "process_result" {
           // Get the payload
           let payload = data["payload"]
           //Merchant handling
         }
     } else {
       //Error handling
     }
})